src/conv.jl

```diff
@@ -188,7 +189,20 @@ for (front_name, backend) in (
         function $(Symbol("$(front_name)!"))(
                         out::AbstractArray{T,5}, in1::AbstractArray{T,5},
                         in2::AbstractArray{T,5}, cdims::ConvDims; kwargs...) where {T <: $G}
-            $(Symbol("$(front_name)_$(backend)!"))(out, in1, in2, cdims; kwargs...)
+            x_cs = Iterators.partition(1:size(in1, 4),
+                                       channels_in(cdims) ÷ groupcount(cdims))
+            w_cs = Iterators.partition(1:size(in2, 5),
+                                       channels_out(cdims) ÷ groupcount(cdims))
+            cdims2 = DenseConvDims(cdims,
+                                   G = 1,
+                                   C_in = channels_in(cdims) ÷ groupcount(cdims),
+                                   C_out = channels_out(cdims) ÷ groupcount(cdims))
+            Threads.@sync Threads.@spawn for (xc, wc) in zip(x_cs, w_cs)
```

Comment:
```suggestion
            Threads.@threads for (xc, wc) in zip(x_cs, w_cs)
```
are these the same?

Reply:
They are different. We are looking to keep the threads as free as possible.

Reply:
> They are different

Yes I checked the docs and saw they are different. You are spawning a single thread carrying over the whole for loop, so in practice this the same as doing it single-threaded, right?

The correct way to parallelize this loop seems to be with `@threads`.

> We are looking to keep the threads as free as possible

I don't know what that means

Reply:
We want to spawn tasks that perform convolutions, which are in turn threaded.

Reply:
I don't know julia's multithreading model very well, so what I'm saying could be wrong but still I give my two cents. 
How things are currently done in this PR is that you spawn a single task running the whole loop, so groups are convolved sequentially. Each group's convolution is then parallelized. I think a better approach is to apply `@threads` to the for loop instead, so that each iteration becomes a taks. Then  tasks and tasks-within-tasks are automatically scheduled by julia.